pkg/apis/toolchain/v1alpha1/masteruserrecord_types.go

```diff
@@ -30,7 +35,16 @@ const (
 	MasterUserRecordDisabledReason                           = disabledReason
 	MasterUserRecordNotificationCRCreatedReason              = "NotificationCRCreated"
 	MasterUserRecordNotificationCRCreationFailedReason       = "NotificationCRCreationFailed"
-	MasterUserRecordUserIDLabelKey                           = LabelKeyPrefix + "user-id"
+
+	// #### LABELS ####
+
+	// MasterUserRecordUserIDLabelKey
+	// Deprecated: This is to be replaced with MasterUserRecordOwnerLabelKey
+	MasterUserRecordUserIDLabelKey = LabelKeyPrefix + "user-id"
+
+	// MasterUserRecordOwnerLabelKey indicates the label value that contains the owner reference for this resource,
+	// which will be the UserSignup instance with the corresponding resource name
+	MasterUserRecordOwnerLabelKey = LabelKeyPrefix + "owner"
```

Comment:
I think we can use a common owner label in all relevant CRDs.

Reply:
I think in this case it's valuable to have the documentation for the constant, indicating exactly where the value comes from.

Reply:
OK. But we still can use a common const for that:
```suggestion
	MasterUserRecordOwnerLabelKey = OwnerLabelKey
```

Reply:
Good point, changed